### PR TITLE
Add tags for assignments (partly fixes Issue #2110)

### PR DESCRIPTION
**Pull request by joelmacht**
Add tags for assignments

### CODE REVIEW

File: openslides/assignments/static/js/assignments/site.js [hunk 1]
```diff
@@ -200,8 +228,16 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
...
                 _.map(assignment.assignment_related_users,
-                        function (candidate) {return candidate.user.get_short_name();}).join(" "),
+                        function (candidate) {return candidate.user.get_short_name();}
+                ).join(" "),
+                _.map(
+                    assignment.tags,
+                    function (tag) {
+                        return tag.name;
+                    }
+                ).join(" "),
```
Comment (emanuelschuetze) [inline openslides/assignments/static/js/assignments/site.js]: Please use consistent code formatting / line breaks (as for line `_map(assignment.assign,ent_related_users...`)

File: openslides/assignments/static/js/assignments/site.js [hunk 2]
```diff
@@ -200,8 +228,16 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
...
             ].join(" ");
+
```
Comment (emanuelschuetze) [inline openslides/assignments/static/js/assignments/site.js]: remove this empty line

**Comment by emanuelschuetze** [comment]
Tested tags for assignments: works good! Only two coding style remarks.

File: openslides/assignments/static/js/assignments/site.js [hunk 3]
```diff
@@ -435,7 +473,8 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
     'Assignment',
     'AssignmentForm',
     'Agenda',
-    function($scope, Assignment, AssignmentForm, Agenda) {
+    'Tag',
```
Comment (normanjaeckel) [inline openslides/assignments/static/js/assignments/site.js]: Why do we need this dependency injection here?

File: openslides/assignments/static/js/assignments/site.js [hunk 4]
```diff
@@ -468,8 +507,9 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
     'Assignment',
     'AssignmentForm',
     'Agenda',
+    'Tag',
```
Comment (normanjaeckel) [inline openslides/assignments/static/js/assignments/site.js]: Why do we need this dependency injection here?

**Comment by normanjaeckel** [comment]
@joelmacht Two more remarks. If you change something, please first add a several commit so I can check the last changes without reviewing everything again. Just before merge we will ask you to rebase your commits. Thanks. ;-)

**Comment by emanuelschuetze** [comment]
@joelmacht Can you fix/answer the two open remarks? Thx.

**Comment by joelmacht** [comment]
@emanuelschuetze Sorry, I was not able to work on this issue for the last weeks because of exams. But I will have to time to fix them in the next two weeks.

**Comment by joelmacht** [comment]
Removed unnecessary injections

File: openslides/assignments/static/js/assignments/site.js [hunk 5]
```diff
@@ -274,7 +310,7 @@ angular.module('OpenSlidesApp.assignments.site', ['OpenSlidesApp.assignments'])
     'User',
     'assignment',
     'phases',
-    function($scope, $http, filterFilter, gettext, ngDialog, AssignmentForm, operator, Assignment, User, assignment, phases) {
+    function($scope, $http, filterFilter, gettext, ngDialog, AssignmentForm, operator, Assignment, User, assignment, Tag, phases) {
```
Comment (normanjaeckel) [inline openslides/assignments/static/js/assignments/site.js]: The `Tag` here is invalid. Please remove it.

File: openslides/assignments/static/templates/assignments/assignment-detail.html [hunk 6]
```diff
@@ -53,6 +53,13 @@ <h3 translate>Number of posts to be elected</h3>
           {{ assignment.open_posts }}<br>
       </div>
       <div class="col-md-6">
```
Comment (normanjaeckel) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: @emanuelschuetze Please check template here. We have three times `col-md-6`. This should be `col-md-4`, shouldn't it?

**Comment by normanjaeckel** [comment]
@joelmacht Sorry. One remark again. Thanks for your work.

File: openslides/assignments/static/templates/assignments/assignment-detail.html (hunk 6 above)
Comment (emanuelschuetze) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: @joelmacht Please use a 2-column layout (2 times col-md-6) and move the tags below the 'posts'. After your change I'll test it.

**Comment by joelmacht** [comment]
Fixed remarks

File: openslides/assignments/static/templates/assignments/assignment-detail.html [hunk 7]
```diff
@@ -68,6 +68,15 @@ <h3 translate>Phase</h3>
           </div>
       </div>
     </div>
+    <div class="row">
+      <div class="col-md-6">
+        <!-- Tags -->
```
Comment (emanuelschuetze) [inline openslides/assignments/static/templates/assignments/assignment-detail.html]: Please move the whole tags block to line 54 (below 'posts' in the first column of col-md-6).

**Comment by emanuelschuetze** [comment]
One last layout remark.

**Comment by joelmacht** [comment]
Updated

**Comment by emanuelschuetze** [comment]
Ok. Looks good now. Please rebase to one commit that we can merge it.

**Comment by joelmacht** [comment]
Rebase done.

**Pull request merged**